### DISCUSSION TITLE
Add a repoRootDir template field

### DISCUSSION THREAD

**Discussion by namrog84** [renovatebot/renovate#24848] [Suggest an Idea]
### What would you like Renovate to be able to do?

in postUpgradeTaskswe can do things with `cd {{packageFileDir}}` and `{{parentDir}}`
but in some instances we'd like to jump back to the root.

it'd be great if there was a `repoRootDir` field available,  Unless there is another already existing easy way to get back to repo root?

### If you have any ideas on how this should be implemented, please tell us here.

Since we already have `packageFileDir`, the code should likely be similar to that but less depth?

### Is this a feature you are interested in implementing yourself?

No

**Comment by rarkins** [comment]
postUpgradeTasks are run in that dir by default if you don't change out of it then that's where you are already. If you do change out of it then you should be able to run `cd -` to get back to where you started from (this is a Linux trick, won't work on Windows).

**Comment by namrog84** [comment]
Good to know.
I think our renovatebot is running on mac for `pod install` related changes.

I know there is things like pushd and popd and various other ways to do the equivalent on windows. Though it'd still be great to be able to just cd directly into the particular renovating repo's root.(if it is at all easily possible, if not I can close this ask now)

**Comment by rarkins** [comment]
It can be done
